### DISCUSSION TITLE
FreeRTOS tick on ESP32

### DISCUSSION THREAD

**Discussion by Villaden74** [Mair/esp32-course#167] [Course topic requests]
Hi,
I'm trying to understand which is the interrupt source (handler) for FreeRTOS time tick.
On ARM I use the core SysTick interrupt source.
On the web I found https://www.esp32.com/viewtopic.php?t=18926. This topic says that an internal CPU timer is used for, but nothing more....
I tryed on official ESP32 documentation, but nothing...
Anyone could give me some tips about ESP32 FreeRTOS time tick management (which interrupt source uses)?
Thanks.

**Comment by noweare1** [comment]
This question was answered on the esp32 forum a few days ago.
